pkg/base1/cockpit.css

```diff
@@ -742,6 +742,20 @@ tbody.active .listing-head {
     clear: right;
 }
 
+div.listing-maybe .listing-error {
```

Comment:
And this is worth merging early as a separate pull request, so others can make use of it.